### PR TITLE
Separate windows build instructions

### PR DESCRIPTION
**Pull request by 0leks**
Added separate BUILD_Windows.md file with instructions for building from source specific to windows.

### CODE REVIEW

File: BUILD_Windows.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+It is recommended **not** to add Anaconda to your PATH and **not** to register Anaconda as your default Python.  
+Using the *Anaconda Prompt*, run `conda_installs.bat`  
+
+### Qt5  
+Download and install the latest version of [[Qt5]](https://download.qt.io/archive/qt/), selecting the LGPL (free) license. (at least version 5.10 required)  
+After installing Qt5, add the directory containing `qmake.exe` to your PATH. (See [Adding to PATH](GettingStarted.md#PATH-environment-variable) for help with this)  
+Example qmake directory: `D:\Qt\5.14.0\winrt_x64_msvc2017\bin`  
+
+### VXL, VTK, ITK, and Eigen
+These three dependencies can be installed using the **build_dependencies.sh** script.
```
Comment (akenmorris) [inline BUILD_Windows.md]: "four" dependencies?

Comment (0leks) [reply BUILD_Windows.md]: good call

**Comment by akenmorris** [comment]
Now that #419 has been merged, we need to update these instructions to activate shapeworks conda environment before running CMake (the github action now does this), or it will fail to find gtest.

I'm not sure how to activate a conda environment for a double-clicked gui application like cmake-gui.exe.  @cchriste any suggestions?  Other applications that run CMake (from scratch) like Qt Creator will have the same problem.

@cchriste, please update the regular BUILD.md for your changes to the build system (e.g. must activate conda, and deps.txt is out of date as well now, or remove the part about "if not using anaconda)).

**Comment by cchriste** [comment]
Thanks Alan. Do you mind creating a separate issue for this?

**Pull request merged**